### PR TITLE
 [V4] Feat: Create wallet

### PR DESCRIPTION
**Pull request by Gamboster**
- Create page improved: Creation of BCH wallets
- Home shows BCH wallets
- To do: Invitation view for shared wallets

### CODE REVIEW

**Comment by Gamboster** [comment]
copay_-_copay_bitcoin_wallet

File: src/pages/add/create-wallet/create-wallet.ts [hunk 1]
```diff
@@ -14,41 +22,70 @@ import * as _ from 'lodash';
...
+    10: 2,
+    11: 1,
+    12: 1,
+  };
+
+  private appName: string;
+  private createForm: FormGroup;
+  private defaults: any;
+  private config: any;
+  private tc: number;
```
Comment (JDonadio) [inline src/pages/add/create-wallet/create-wallet.ts]: totalCopayers?

File: src/pages/add/create-wallet/create-wallet.html [hunk 2]
```diff
@@ -19,19 +19,27 @@
...
         <ion-label stacked>Required number of signatures</ion-label>
-        <ion-select [(ngModel)]="formData.requiredCopayers" formControlName="requiredCopayers" (ionChange)="signaturesChange(formData.requiredCopayers)">
+        <ion-select [(ngModel)]="formData.requiredCopayers" formControlName="requiredCopayers">
           <ion-option *ngFor="let signature of signatures" [value]="signature">{{signature}}</ion-option>
         </ion-select>
       </ion-item>
     </div>
 
+    <ion-item>
+      <ion-label stacked>Coin</ion-label>
```
Comment (JDonadio) [inline src/pages/add/create-wallet/create-wallet.html]: Missing translate

File: src/pages/home/home.ts [hunk 3]
```diff
@@ -62,11 +80,14 @@ export class HomePage {
       })
   }
 
-  public goToAddView(): void {
-    this.navCtrl.push(AddPage);
+  public goToAddView(coin?: string): void {
+    if (coin)
```
Comment (JDonadio) [inline src/pages/home/home.ts]: Do not need to use IF sentence

**Pull request merged**